src/corelibs/U2View/src/ov_msa/mca_reads/McaReadsTabFactory.cpp

```diff
@@ -0,0 +1,71 @@
+/**
+ * UGENE - Integrated Bioinformatics Tools.
+ * Copyright (C) 2008-2021 UniPro <ugene@unipro.ru>
+ * http://ugene.net
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+ * MA 02110-1301, USA.
+ */
+
+#include "McaReadsTabFactory.h"
+
+#include "McaAlternativeMutationsWidget.h"
+
+#include <U2Core/U2SafePoints.h>
+#include <U2View/MSAEditor.h>
+#include <U2Gui/ShowHideSubgroupWidget.h>
+
+#include <QVBoxLayout>
+
+namespace U2 {
+
+const QString McaReadsTabFactory::GROUP_ID = "OP_MCA_READS";
+const QString McaReadsTabFactory::GROUP_ICON_STR = ":core/images/todo.png";
+const QString McaReadsTabFactory::GROUP_DOC_PAGE = "TODO";
+
+McaReadsTabFactory::McaReadsTabFactory() {
+    objectViewOfWidget = ObjViewType_ChromAlignmentEditor;
+}
+
+QWidget* McaReadsTabFactory::createWidget(GObjectView* objView, const QVariantMap& options) {
+    Q_UNUSED(options);
```

Comment:
nit: if you do not use this param just do not give it a name (Q_UNUSED is not needed in this case)

Reply:
Well, I already told about it - we don't have any agreements about it and I prefer use `Q_UNUSED`. Also there's the attribute `[[maybe_unused]]` from C++17  for these cases, so either we make the agreement or anyone can use the way they like the most

Reply:
Actually, I think that the best salvation for this point is `maybe_unused`, because it's the solution which was specially developed for these cases on the low-level of C++ itself (not some hand-made macro like `Q_UNUSED` or a trick like the way you use). But I'm not sure that we can use it (cuz it's from C++17), so I prefer `Q_UNUSED` because it seems more similar to `Q_UNUSED`.

Reply:
If the var is not used we can omit it: this is a feature of C++ intentionally designed for it and available even in the old standard. Why not using the standard feature and wait for some fancy future standard?

Reply:
You wont change my opinion about this. If you insist on using your way, I will do it

Reply:
OK